spark/src/main/scala/vegas/sparkExt/package.scala

```diff
@@ -1,16 +1,14 @@
-package vegas.spark
+package vegas
 
 import org.apache.spark.rdd.RDD
 import org.apache.spark.sql.DataFrame
-
-import vegas.DSL._
 import vegas.DSL.DataDSL
 
-object Spark {
+package object sparkExt {
```

Comment:
yeah. I had to rename it because in Spark 2.0, they no have a implicit variable `spark`. So when we import `vegas._` it conflicts.

So import will now need to be
```scala
import vegas.sparkExt._
```